configure.ac

```diff
@@ -199,6 +196,12 @@ if test "$with_c_glib" = "yes"; then
 fi
 AM_CONDITIONAL(WITH_C_GLIB, [test "$have_glib2" = "yes" -a "$have_gobject2" = "yes"])
 
+echo "OpenSSL check"
```

Comment:
There may be other libraries with native builds that need OpenSSL.  Why are we making it conditional on cpp and c_glib library support?